.github/workflows/changelog-assembly.yml

```diff
@@ -21,9 +32,15 @@ jobs:
           crate: aeruginous
 
       - run: |
-          aeruginous ronlog release \
-            -i changelog.d/ \
-            -v "$(cargo r -- --version | cut -d\  -f2)"
+          aeruginous increment-version \
+            -v "$(cat .version)" \
+            -r ${{ inputs.release }} \
+            -e .version \
+            -e CITATION.cff \
+            -e Cargo.lock \
+            -e Cargo.toml
+          aeruginous cff-release-today CITATION.cff
+          aeruginous ronlog release -i changelog.d/ -v "$(cat .version)"
```

Comment:
And finally, we assemble our RONLOG.  This step is now easier because we have the version number in a configuration file such that this workflow should succeed faster.